### PR TITLE
BAH-2181 | Added env for openmrs container

### PR DESCRIPTION
**Pull request by atish160384**
- Added few environment variables for openmrs container to be fetched in appointments module

### CODE REVIEW

File: bahmni-lite/.env [hunk 1]
```diff
@@ -9,6 +9,8 @@ OPENMRS_HOST=openmrs
 OPENMRS_PORT=8080
 OPENMRS_ATOMFEED_USER=admin
 OPENMRS_ATOMFEED_PASSWORD=Admin123
+OPENMRS_SYSTEM_SMS_USER=systemuser
```
Comment (gsluthra) [inline bahmni-lite/.env:12]: Can we plz make this user as `system-user`.

**Comment by mohan-13** [comment]
Hello, I would like to get a bit of context here.  Why does the OpenMRS container in itself needs the HOST and PORT parameter ? Why do we need credentials from within the OMRS application context itself?

File: bahmni-lite/.env (hunk 1 above)
Comment (atish160384) [reply bahmni-lite/.env:12]: Sure will do the changes

**Comment by atish160384** [comment]
> Hello, I would like to get a bit of context here. Why does the OpenMRS container in itself needs the HOST and PORT parameter ? Why do we need credentials from within the OMRS application context itself?

So Mohan I had added them for scheduler which needs to send message to the patients having appointment for sending the message we need a reporting_session cookie to (in sms-service) and to extract that reporting session (of system-user) we would need to Authenticate and login to openmrs first for which I added these Host and Port

**Pull request closed without merging**